plugins/inventory/libvirt.py

```diff
@@ -1,18 +1,18 @@
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 from __future__ import (absolute_import, division, print_function)
 __metaclass__ = type
 
 DOCUMENTATION = r'''
 name: libvirt
-plugin_type: inventory
```

Comment:
@odyssey4me do you mind checking whether me removing `plugin_type` is the right thing to do here? Thanks!

Reply:
Best to get advice from the core developers on this one. Perhaps something has changed, but this was originally a requirement - maybe they've reduced the friction a little bit.

Reply:
Will, do thanks @odyssey4me!

Reply:
And removal of `plugin_type` is confirmed here https://github.com/ansible-collections/news-for-maintainers/discussions/11#discussioncomment-2296426